### PR TITLE
change modal class to something more specific so styles aren't easily…

### PR DESCRIPTION
**Pull request by will-quanstrom**
… overwritten by themes

### CODE REVIEW

File: src/widgets.ts [hunk 1]
```diff
@@ -19,6 +19,9 @@ function loadExpAppWidget(keyAttribute: string, scriptPath: string): void {
     document.body.appendChild(script);
   }
 }
-const baseUrl = "__BASE_URL__/public/__ENV_NAME__";
+// baseUrl for Development
+const baseUrl = "__BASE_URL__/__ENV_NAME__";
```
Comment (ckeyes88) [inline src/widgets.ts:23]: Please remove these changes as they're needed for deployment.

File: example/index.html [hunk 2]
```diff
@@ -6,17 +6,95 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Event Gallery Widget Dev</title>
-  <!-- <link href="/dist/calendarBookingForm.css" rel="stylesheet" /> -->
-  <link href="/dist/expAppListView.css" rel="stylesheet" />
+  <!-- <link href="/dist/calendarBookingForm.css" rel="stylesheet" />
+  <link href="/dist/expAppListView.css" rel="stylesheet" /> -->
 
 </head>
+asdfdsf<br>
```
Comment (ckeyes88) [inline example/index.html:13]: uhhhh what the heck is going on here haha?

File: src/widgets.ts [hunk 3]
```diff
@@ -21,7 +21,7 @@ function loadExpAppWidget(keyAttribute: string, scriptPath: string): void {
 }
 // baseUrl for Development
 const baseUrl = "__BASE_URL__/__ENV_NAME__";
-
+// baseURL for Production
 // const baseUrl = "__BASE_URL__/public/__ENV_NAME__";
```
Comment (ckeyes88) [inline src/widgets.ts:25]: this still needs to be `__BASE_URL__/public/__ENV_NAME__`

File: src/calendarBookingForm/Components/Portal.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import { VNode, Component } from "preact";
```
Comment (ckeyes88) [inline src/calendarBookingForm/Components/Portal.tsx:1]: I'm not super familiar with react portals. Can you explain why you used this pattern and what it's accomplishing for you. I'm not saying it's wrong. I just want to make sure you understand what it's doing and why it was necessary.

Comment (ckeyes88) [reply src/calendarBookingForm/Components/Portal.tsx:1]: ignore this comment. I just found out this came from Nick.

File: example/index.html (hunk 2 above)
Comment (will-quanstrom) [reply example/index.html:13]: Yeahhhhh

File: src/widgets.ts (hunk 3 above)
Comment (will-quanstrom) [reply src/widgets.ts:25]: Sorry I push up changes without thinking that they were changing this PR I already put in. Lesson learned.

**Pull request merged**